### DISCUSSION TITLE
guide/drivers/thunder

### DISCUSSION THREAD

**Discussion by giscus** [AlistGo/docs#274] [Announcements]
# guide/drivers/thunder

Please use Thunder directly instead of ThunderExpert if you are not good at it. ThunderExpert mainly provides more free settings and realizes more login methods Thunder username...

https://alist.nn.ci/guide/drivers/thunder.html

**Comment by WonderWu123** [comment]
迅雷验证页面失效了，显示captcha_invalid

**Comment by hiosh** [comment]
我正确地在验证码的网站进行了短信验证，也把对应的token粘到了登录框里，可是却收到4002的错误提示，说是我的手机号前面有+86，这是啥情况？然后我如果在id里加上了“+86”，又提示我密码错误。。。
